{"uid":"c85be2181c3b4fc2","name":"Авторизация пользователя с не валидными данными","fullName":"tests.api.auth.test_auth_api.TestAuth#test_user_login","historyId":"e1b675872cdfea4013982e1ce2bf432f","time":{"start":1754561116659,"stop":1754561116708,"duration":49},"status":"failed","statusMessage":"AssertionError: Не верный статус код. exp:400, act:401","statusTrace":"self = <test_auth_api.TestAuth object at 0x7f42af78f860>\n\n    @allure.title(\"Авторизация пользователя с не валидными данными\")\n    @allure.testcase(\"ID-171\")\n    def test_user_login(self):\n        user_data = AuthNormalRequestModel(username=\"test\", password=\"123456\")\n        response = public_users_client().auth(user_data)\n    \n>       assert_status_code(HTTPStatus.BAD_REQUEST, response.status_code)\n\ntests/api/auth/test_auth_api.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nexpected = <HTTPStatus.BAD_REQUEST: 400>, actual = 401\n\n    @allure.step(\"Проверка статус код равен {expected}\")\n    def assert_status_code(expected: int, actual: int) -> None:\n        \"\"\"\"\n        Проверяет, что фактический статус-код ответа соответствует ожидаемому.\n    \n        :param expected: Ожидаемый статус-код.\n        :param actual: Фактический статус-код ответа.\n        :raises AssertionError: Если статус-коды не совпадают.\n        \"\"\"\n        logger.info(f\"Check that response status code equals to {expected}\")\n>       assert actual == expected, f\"Не верный статус код. exp:{expected}, act:{actual}\"\n               ^^^^^^^^^^^^^^^^^^\nE       AssertionError: Не верный статус код. exp:400, act:401\n\nutils/asserts.py:22: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1754561116658,"stop":1754561116658,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1754561116658,"stop":1754561116658,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1754561116658,"stop":1754561116658,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1754561116658,"stop":1754561116658,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Не верный статус код. exp:400, act:401","statusTrace":"self = <test_auth_api.TestAuth object at 0x7f42af78f860>\n\n    @allure.title(\"Авторизация пользователя с не валидными данными\")\n    @allure.testcase(\"ID-171\")\n    def test_user_login(self):\n        user_data = AuthNormalRequestModel(username=\"test\", password=\"123456\")\n        response = public_users_client().auth(user_data)\n    \n>       assert_status_code(HTTPStatus.BAD_REQUEST, response.status_code)\n\ntests/api/auth/test_auth_api.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nexpected = <HTTPStatus.BAD_REQUEST: 400>, actual = 401\n\n    @allure.step(\"Проверка статус код равен {expected}\")\n    def assert_status_code(expected: int, actual: int) -> None:\n        \"\"\"\"\n        Проверяет, что фактический статус-код ответа соответствует ожидаемому.\n    \n        :param expected: Ожидаемый статус-код.\n        :param actual: Фактический статус-код ответа.\n        :raises AssertionError: Если статус-коды не совпадают.\n        \"\"\"\n        logger.info(f\"Check that response status code equals to {expected}\")\n>       assert actual == expected, f\"Не верный статус код. exp:{expected}, act:{actual}\"\n               ^^^^^^^^^^^^^^^^^^\nE       AssertionError: Не верный статус код. exp:400, act:401\n\nutils/asserts.py:22: AssertionError","steps":[{"name":"Выполнение авторизации пользователя","time":{"start":1754561116681,"stop":1754561116698,"duration":17},"status":"passed","steps":[{"name":"Выполнение POST-Запроса","time":{"start":1754561116681,"stop":1754561116698,"duration":17},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"url","value":"'/auth'"},{"name":"json","value":"{'username': 'test', 'type': 'normal', 'password': '123456'}"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"user_data","value":"AuthNormalRequestModel(username='test', type='normal', password='123456')"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Проверка статус код равен <HTTPStatus.BAD_REQUEST: 400>","time":{"start":1754561116698,"stop":1754561116698,"duration":0},"status":"failed","statusMessage":"AssertionError: Не верный статус код. exp:400, act:401\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/pet-project-autotests/pet-project-autotests/utils/asserts.py\", line 22, in assert_status_code\n    assert actual == expected, f\"Не верный статус код. exp:{expected}, act:{actual}\"\n           ^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[{"name":"expected","value":"<HTTPStatus.BAD_REQUEST: 400>"},{"name":"actual","value":"401"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"32396c5a0e7454ed","name":"log","source":"32396c5a0e7454ed.txt","type":"text/plain","size":273},{"uid":"6d612e55dc2cad16","name":"stderr","source":"6d612e55dc2cad16.txt","type":"text/plain","size":302}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Пользователи"},{"name":"feature","value":"Auth"},{"name":"tag","value":"users"},{"name":"tag","value":"auth"},{"name":"tag","value":"api"},{"name":"parentSuite","value":"tests.api.auth"},{"name":"suite","value":"test_auth_api"},{"name":"subSuite","value":"TestAuth"},{"name":"host","value":"pkrvmjbmru5nbw0"},{"name":"thread","value":"6759-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.api.auth.test_auth_api"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"ID-171","url":"ID-171","type":"tms"}],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["auth","api","users"]},"source":"c85be2181c3b4fc2.json","parameterValues":[]}